pkg/kubelet/kubelet_getters.go

```diff
@@ -120,8 +120,10 @@ func (kl *Kubelet) ListPodsFromDisk() ([]types.UID, error) {
 // user namespaces.
 func (kl *Kubelet) HandlerSupportsUserNamespaces(rtHandler string) (bool, error) {
 	rtHandlers := kl.runtimeState.runtimeHandlers()
-	if rtHandlers == nil {
-		return false, fmt.Errorf("runtime handlers are not set")
+	if len(rtHandlers) == 0 {
+		// The slice is empty if the runtime is old and doesn't support this message.
+		// In this case, the runtime is too old to support user namespaces.
```

Comment:
nit: the second line in the comment seems redundant to me
question: when this was supposed to return nil? can it ever return nil, or was it a mistake from the beginning?

Reply:
I don't know if the behavior changed, it might, but this message is not implemented and I don't think we should depend on it retuning nil vs an empty non-nil value. So this is the right way to do it, IMHO :)